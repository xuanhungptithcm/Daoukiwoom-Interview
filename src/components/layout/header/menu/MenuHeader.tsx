import { ReactElement } from "react";
import { ReactComponent as LogoSVG } from "../../../../assets/icons/svg/logo.svg";
import { ReactComponent as HomeIconSVG } from "../../../../assets/icons/svg/home.svg";
import "./menuHeader.scss";
import { Link } from "react-router-dom";

const MenuHeader = (): ReactElement => {
  return (
    <div className="container-menu-header">
      <div className="menu-header">
        <div className="logo">
          <Link to="/">
            <LogoSVG />
          </Link>
        </div>
        <ul className="menus">
          <li className="menu-item active">
            <HomeIconSVG />
            <a href="/">Home</a>
          </li>
          <li className="menu-item">
            <a href="/home">Auto Homepage</a>
          </li>
          <li className="menu-item">
            <a href="/sales">Sales</a>
          </li>
          <li className="menu-item">
            <a href="/cmr">CRM</a>
          </li>
          <li className="menu-item">
            <a href="/electronic-office">Electronic office</a>
          </li>
          <li className="menu-item">
            <a href="/cloud-erp">Cloud ERP</a>
          </li>
          <li className="menu-item">
            <a href="/ssl">SSL</a>
          </li>
        </ul>
      </div>
    </div>
  );
};

export default MenuHeader;
