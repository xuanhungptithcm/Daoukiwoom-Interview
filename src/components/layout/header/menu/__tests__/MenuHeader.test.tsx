import { shallow } from "enzyme";
import MenuHeader from "../MenuHeader";

describe("MenuHeader", () => {
  test("should properly render", () => {
    expect(shallow(<MenuHeader />)).toMatchSnapshot();
  });
});
