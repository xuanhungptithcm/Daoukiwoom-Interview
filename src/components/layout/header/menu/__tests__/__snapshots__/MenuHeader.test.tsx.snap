// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuHeader should properly render 1`] = `
<div
  className="container-menu-header"
>
  <div
    className="menu-header"
  >
    <div
      className="logo"
    >
      <Link
        to="/"
      >
        <ReactComponent />
      </Link>
    </div>
    <ul
      className="menus"
    >
      <li
        className="menu-item active"
      >
        <ReactComponent />
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li
        className="menu-item"
      >
        <a
          href="/home"
        >
          Auto Homepage
        </a>
      </li>
      <li
        className="menu-item"
      >
        <a
          href="/sales"
        >
          Sales
        </a>
      </li>
      <li
        className="menu-item"
      >
        <a
          href="/cmr"
        >
          CRM
        </a>
      </li>
      <li
        className="menu-item"
      >
        <a
          href="/electronic-office"
        >
          Electronic office
        </a>
      </li>
      <li
        className="menu-item"
      >
        <a
          href="/cloud-erp"
        >
          Cloud ERP
        </a>
      </li>
      <li
        className="menu-item"
      >
        <a
          href="/ssl"
        >
          SSL
        </a>
      </li>
    </ul>
  </div>
</div>
`;
